web/e2e-tests/message-basics.test.ts

```diff
@@ -142,7 +142,7 @@ async function search_and_check(
     check: (page: Page) => Promise<void>,
     expected_narrow_title: string,
 ): Promise<void> {
-    await page.click(".search_icon");
+    await page.click(".search_button");
```

Comment:
I'll separate out a commit for switching the container to a button and naming it such to be less confusing